lib/decimal.ex

```diff
@@ -70,8 +72,10 @@ defmodule Decimal do
   @type exponent :: integer
 
   @typedoc """
+
     * `1` for positive
     * `-1` for negative
+
```

Comment:
Actually, I see that you added this almost everywhere. I don't think the trailing whitespace line adds any value but if we should have it we should probably add it to the style guide?